src/ui/setup/index.ts

```diff
@@ -50,6 +52,18 @@ export function bootstrapApp() {
     store.dispatch(setModal("workspace-settings"));
   }
 
+  // Check if we have any preferences saved for this replay and apply
+  // them to the store
+  asyncStore.replaySessions.then((sessions: ReplaySessions) => {
```

Comment:
this would probably be better done before we create the store. that way we can pass it in as initial state. look at how we do it with pending breakpoints.